phpBB/styles/prosilver/template/simple_footer.html

```diff
@@ -8,8 +8,9 @@
 	<div id="darkenwrapper" class="darkenwrapper" data-ajax-error-title="{L_AJAX_ERROR_TITLE}" data-ajax-error-text="{L_AJAX_ERROR_TEXT}" data-ajax-error-text-abort="{L_AJAX_ERROR_TEXT_ABORT}" data-ajax-error-text-timeout="{L_AJAX_ERROR_TEXT_TIMEOUT}" data-ajax-error-text-parsererror="{L_AJAX_ERROR_TEXT_PARSERERROR}">
 		<div id="darken" class="darken">&nbsp;</div>
 	</div>
-	<div id="loading_indicator" class="loading_indicator"></div>
-
+	<div id="loading_indicator" class="loading_indicator">
+		<div class="loader"></div>
```

Comment:
Do we need an alt text for the loading image/thing? Empty elements are always so ugly :/